File: middlePro/src/model/mdb/bot_setting.go
package mdb

type BotSettingModel struct {
	ID        int    `gorm:"column:id;primary_key" json:"id"`
	Tgtoken   string `gorm:"column:tgtoken" json:"tgtoken"`
	Belongs   int    `gorm:"column:belongs" json:"belongs"`
	Cname     string `gorm:"column:cname" json:"cname"`
	Cusername string `gorm:"column:cusername" json:"cusername"`
	MechId    int    `gorm:"column:mech_id" json:"mech_id"`
	Ctype     int    `gorm:"column:ctype" json:"ctype"`
	Sta       int    `gorm:"column:sta" json:"sta"`
}

// TableName sets the insert table name for this struct type
func (c *BotSettingModel) TableName() string {
	return "bot_setting"
}

File: backPro/src/views/vtemplate/botsetting.ts
import { h } from 'vue';

export const columns = [
  {
    title: '密钥token',
    key: 'tgtoken',
    width: 250,
  },
  {
    title: '所属',
    key: 'belongs',
    width: 150, 
      render(row) {
         if(row.belongs == 1) {         
          return "系统所属"
         }else{
           return row.mech_id+"所属"
         }
      }, 
  },
  {
    title: '机器人name',
    key: 'cname',
    width: 150,   
  },
  {
    title: '机器人名字',
    key: 'cusername',
    width: 150,   
  },
  {
    title: '使用类别',
    key: 'ctype',
    width: 180,  
     render(row) {
        if(row.ctype == 1) {         
         return "siglebot使用"
        }else{
          return "群bot使用"
        }
     }, 
  },
  {
    title: '商户ID',
    key: 'mech_id',
    width: 150,   
  },
  
 
  {
    title: '启用',
    key: 'sta',
    width: 50,
    render(row) {
       if(row.sta == 1) {         
        return "正常"
       }else{
         return "禁用"
       }
    },    
  }  
];
